sources/data-node/api/v1/trading_data.proto

```diff
@@ -164,6 +164,12 @@ service TradingDataService {
 
   // Get a specific node by ID
   rpc GetNodeByID(GetNodeByIDRequest) returns (GetNodeByIDResponse);
+  
+  // Get all key rotations
+  rpc GetKeyRotations(GetKeyRotationsRequest) returns (GetKeyRotationsResponse);
+
+  // Get all key rotations by node
+  rpc GetKeyRotationsByNode(GetKeyRotationsByNodeRequest) returns (GetKeyRotationsByNodeResponse);
```

Comment:
Minor: maybe GetKeyRotationsByNode**ID**()?

Reply:
It was called that, but I have changed it to this because this seems to follow pattern in the proto better.

Reply:
For instance: https://github.com/vegaprotocol/protos/blob/f3220ac35af7101ba2a33c9af750c6f8351fbc56/sources/data-node/api/v1/trading_data.proto#L782

Or:
https://github.com/vegaprotocol/protos/blob/f3220ac35af7101ba2a33c9af750c6f8351fbc56/sources/data-node/api/v1/trading_data.proto#L900